modules/internal/DefaultSparse.chpl

```diff
@@ -49,6 +49,10 @@ module DefaultSparse {
       this.dist = dist;
     }
 
+    proc stridable param {
+      return parentDom.stridable;
+    }
+
```

Comment:
I don't mind if this goes in on this PR. The previous behavior was... that `stridable` was unresolved on DefaultSparse?

Reply:
Yes.  And I think this comes up in contexts (like slicing maybe?) where we are applying general recipes in ChapelArray.chpl to arbitrary array/domain implementations.